src/otaclient/ota_core.py

```diff
@@ -579,6 +580,14 @@ def _execute_update(self):
             )
         )
 
+        # NOTE(20250205): for current rebuild-mode, we look at active slot's file_table.
+        # NOTE: get the db via active_slot mount_point
+        _base_ft_db = replace_root(
+            Path(cfg.IMAGE_META_DPATH) / OTAMetadata.FSTABLE_DB,
+            old_root="/",
+            new_root=Path(cfg.ACTIVE_SLOT_MNT),
+        )
+
```

Comment:
change the order in this function(report the status first, then execute the logic).